purchase_request/views/purchase_request_view.xml

```diff
@@ -196,11 +202,6 @@
                                     <field name="date_required" />
                                     <field name="estimated_cost" widget="monetary" />
                                     <field name="currency_id" invisible="1" />
-                                    <field
```

Comment:
Why removing it?

Reply:
What is the purpose of displaying the `company_id` field in the lines? None, let alone allowing to define that field (with something different than what the header has).